docs/platform/getting-started/repository.md

```diff
@@ -30,53 +21,83 @@ dcloud repo -i
 ```
 
 ---
-## GitHub
+### GitHub
 
-### Adding an SSH Key
+#### Adding an SSH Key
 
 - Go to `https://github.com/<accountName>/<repoName>/settings/keys/new`
 - Add your key
 
 ![GitHub SSH access](/img/docs/cloud/github-ssh-key.png)
 
-### Adding a web hook
+#### Adding a web hook
 
 - Go to `https://github.com/<accountName>/<repoName>/settings/hooks/new`
 - Add the webhook provided by `dcloud repo:list -i`.
 
 ![GitHub WebHook](/img/docs/cloud/github-webhook.png)
 
 ---
-## Bitbucket
+### Bitbucket
 
-### Adding an SSH Key
+#### Adding an SSH Key
 
 - Go to `https://bitbucket.org/<accountName>/<repoName>/admin/access-keys/`
 - Add your key
 
 ![BitBucket SSH access - add](/img/docs/cloud/bitbucket-ssh-key-2.png)
 
-### Adding a web hook
+#### Adding a web hook
 
 Coming soon
 
 ---
-## GitLab
+### GitLab
 
-### Adding an SSH Key
+#### Adding an SSH Key
 
 - Go to `https://gitlab.com/<accountName>/<repoName>/-/settings/repository`
 - Open the **Deploy keys** section
 - Add your key
 
 ![GitLab SSH access](/img/docs/cloud/gitlab-ssh-key.png)
 
-### Adding a web hook
+#### Adding a web hook
 
 Coming soon
 
-## Setup Complete
+### Setup Complete
 
 That's everything done. You're now ready to start using Deity Cloud. View the full list of `dcloud` [commands](../cloud/dcloud).
 
 Next step, [deploy](/docs/platform/deployment/overview) your code.
+
+---
+
+## 2. Check your connection
+
+To check your repository and cloud instance are linked run the following command. Your repo URL should be returned
+
+```bash
+dcloud repo
+```
+
+## 3. View your builds
+
+Every time you push a commit that will trigger a build. These builds are not 'deployed' automatically but are made available to deploy once they are finished.
```

Comment:
```suggestion
Every time you push a commit, a build is triggered. These builds are not 'deployed' automatically but are made available to deploy once they are finished.
```
The way it was phrased initially made me read it as `Every time you push (a commit that will trigger a build) ...missing remaining sentence` I.e. as if triggering a build is an optional property of the commit. In the end there was no room for misinterpretation, so what's written here is fine too, just confused me for a split second.